ansible/config/linux/docker_dnsmasq_conf

```diff
@@ -0,0 +1,19 @@
+# Set dns forwarder.
+server=/loc/172.17.42.1
```

Comment:
I set the whole bunch on 172.17.0.x address space, which seems to be the new standard on native linux for docker interface since 1.10.